### DISCUSSION TITLE
Which epoch's results to report? (best val or final)

### DISCUSSION THREAD

**Discussion by sunjin-k** [snap-stanford/ogb#362] [General]
Hello, 

Can I report the test accuracy at the epoch when the validation accuracy was highest? Or do I have to report the test accuracy at the final epoch?

Best regards,
Sunjin

**Comment by weihua916** [comment]
We usually report the test accuracy at the best val epoch.

**Reply by sunjin-k** [reply]
Thanks a lot Weihua!
